### ISSUE TITLE
Html attribute is truncated

### ISSUE THREAD

**Issue by hlecorche** [dzango/TwigTruncateExtension#2]
Hello,

The truncate function breaks html attributes :

For exemple:

```
{{ '<div style="color: red;">Hello world</div>'|truncate(5) }}
```

display:

```
<div style="color:</div>...
```

style attribute is not closed and it's truncated

**Comment by opichon** [comment]
Indeed, it looks like the original truncate function (which this extension is just a wrapper for) did not support this. I'll try and fix this as soon as possible.
